pkg/porter/build_integration_test.go

```diff
@@ -230,3 +230,24 @@ func TestBuildOptions_Defaults(t *testing.T) {
 		assert.Equal(t, config.BuildDriverDocker, opts.Driver)
 	})
 }
+
+func TestPorter_BuildWithCustomValues(t *testing.T) {
+	p := NewTestPorter(t)
+	defer p.Teardown()
+
+	p.TestConfig.TestContext.AddTestFile("./testdata/porter-with-custom-values.yaml", config.Name)
```

Comment:
Instead of making a new test porter.yaml file for this test, how about we edit an existing testdata porter.yaml file and add in the custom section? I've learned the hard way that having lots of test porter.yaml files is a lot of work to maintain any time we change the format of the file.